src/Compilers/CSharp/Portable/Binder/InMethodBinder.cs

```diff
@@ -202,8 +202,8 @@ internal override TypeWithAnnotations GetIteratorElementType(YieldStatementSynta
                         return (((NamedTypeSymbol)returnType).TypeArgumentsWithAnnotationsNoUseSiteDiagnostics[0], false);
                 }
 
-                if (TypeSymbol.Equals(originalDefinition, compilation.GetWellKnownType(WellKnownType.System_Collections_Generic_IAsyncEnumerable_T), TypeCompareKind.ConsiderEverything2) ||
-                    TypeSymbol.Equals(originalDefinition, compilation.GetWellKnownType(WellKnownType.System_Collections_Generic_IAsyncEnumerator_T), TypeCompareKind.ConsiderEverything2))
+                if (TypeSymbol.Equals(originalDefinition, compilation.GetWellKnownType(WellKnownType.System_Collections_Generic_IAsyncEnumerable_T, recordUsage: false), TypeCompareKind.ConsiderEverything2) ||
```

Comment:
>ConsiderEverything2 [](start = 185, length = 19)

nit: ConsiderEverything

Reply:
> nit: ConsiderEverything

Not related to the change

---
In reply to: [350911212](https://github.com/dotnet/roslyn/pull/40010#discussion_r350911212) [](ancestors = 350911212)